docs/setup/mac-setup-docker.md

```
@@ -46,21 +52,31 @@ This command works on both Intel Macs (which use `/usr/local`) and Apple Silicon
 > [!NOTE]
 > The `uv` package manager is installed via Homebrew in the previous step. It will be used to manage Python dependencies and virtual environments for the Cinemata project. This is a faster alternative to `pip` and provides better dependency resolution.
 
+CinemataCMS requires a minimum `npm` version 11. Enter the following to check your `npm` version:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)